components/rewards/Transaction.js

```diff
@@ -11,30 +12,77 @@ import CircleIcon from '../CircleIcon';
  * */
 
 function Transaction(props) {
-  const { date, storeName, pointsEarned, totalSale } = props;
+  const {
+    date,
+    storeName,
+    pointsEarned,
+    totalSale,
+    rewardsUnlocked,
+    rewardsApplied,
+  } = props;
   const options = {
     weekday: 'short',
     year: 'numeric',
     month: 'short',
     day: 'numeric',
   };
   return (
-    <TransactionCard>
-      <CircleIcon
-        icon="check"
-        iconColor={Colors.primaryGreen}
-        circleColor={Colors.lightestGreen}
-      />
-      <ContentContainer>
-        <Caption color={Colors.secondaryText}>
-          {`${date.toLocaleDateString('en-US', options)} • ${storeName}`}
-        </Caption>
-        <Subhead>{`${pointsEarned} points earned`}</Subhead>
-        <Caption color={Colors.secondaryText}>
-          {`for ${displayDollarValue(totalSale || 0)} of healthy products`}
-        </Caption>
-      </ContentContainer>
-    </TransactionCard>
+    <View>
+      {/* Display points earned */}
+      <TransactionCard>
+        <CircleIcon
+          icon="check"
+          iconColor={Colors.primaryGreen}
+          circleColor={Colors.lightestGreen}
+        />
+        <ContentContainer>
+          <Caption color={Colors.secondaryText}>
+            {`${date.toLocaleDateString('en-US', options)} • ${storeName}`}
+          </Caption>
+          <Subhead>{`${pointsEarned} points earned`}</Subhead>
+          <Caption color={Colors.secondaryText}>
+            {`for ${displayDollarValue(totalSale || 0)} of healthy products`}
```

Comment:
I know it was like this before but would this make more sense to be subtotal?

Reply:
Oh nevermind I guess it makes sense that the number of points directly corresponds to the total sale (bc the rewards points are deducted?) but this language might not fit since they did actually pay more for the total healthy products...if that makes sense
cc @12aschen ?

Reply:
It's probably easiest to match what we do in Clerk (which I cannot currently recall) but I would guess this is a quacetion that we can defer. Just don't forget to check in on it 😅